pygmt/src/solar.py

```diff
@@ -36,11 +45,11 @@ def solar(self, terminator="d", terminator_datetime=None, **kwargs):
 
     Parameters
     ----------
-    terminator : str
+    terminator
         Set the type of terminator displayed. Valid arguments are
         ``"day_night"``, ``"civil"``, ``"nautical"``, and ``"astronomical"``,
         which can be set with either the full name or the first letter of the
```

Comment:
Now the available types of terminators are already documented within the function string. So we can maybe shorten this docstring to:

```suggestion
    terminator
        Set the type of terminator displayed, which can be set with either
        the full name or the first letter of the name.
```

Reply:
Similar to the `data_source` parameter of the `load_earth_relief` function, I've written the description to:

```
   terminator
        Set the type of terminator displayed, which can be set with either the
        full name or the first letter of the name. Available options are:
        - ``"astronomical"``: Astronomical twilight
        - ``"civil"``: Civil twilight
        - ``"day_night"``: Day/night terminator
        - ``"nautical"``: Nautical twilight
        Refer to https://en.wikipedia.org/wiki/Twilight for the definitions of
        different types of twilight.
```